modules/storage-rocksdb/src/main/java/org/apache/ignite/internal/storage/rocksdb/GarbageCollector.java

```diff
@@ -205,7 +205,10 @@ boolean tryAddToGcQueue(WriteBatchWithIndex writeBatch, RowId rowId, HybridTimes
         // However, the element that we need to garbage collect is the next (older one) element.
         // First we check if there's anything to garbage collect. If the element is a tombstone we remove it.
         // If the next element exists, that should be the element that we want to garbage collect.
-        try (RocksIterator gcIt = db.newIterator(gcQueueCf, helper.upperBoundReadOpts)) {
+        try (
+                RocksIterator newGcIt = db.newIterator(gcQueueCf, helper.upperBoundReadOpts);
```

Comment:
This pattern can be seen a lot of times in this code, I think it should be extracted into a separate method

Reply:
Try to fix it.